### DISCUSSION TITLE
Select all when editing a value on a node

### DISCUSSION THREAD

**Discussion by GrimPixel** [IndiePython/nodezator#74] [Feature Request]
Currently, when I click on the value, the focus is at the end of the string. It is less likely that the user would like to append numbers to it and more likely to write a new one.

**Comment by KennedyRichard** [comment]
I agree. This is also how other apps do it.

This will be the default behavior when text selection is implemented, because the entire content will appear selected and the user will just have to start typing so the content is overridden right away. Since text selection is not implemented yet, for now you can use the keys Ctrl + Backspace to delete the content behind the current position.
